apps/server/src/modules/system/controller/dto/system.response.ts

```diff
@@ -0,0 +1,40 @@
+import { OauthConfigResponse } from '@src/modules/system/controller/dto/oauth-config.response';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class SystemResponse {
+	constructor(system: SystemResponse) {
+		this.type = system.type;
+		this.url = system.url;
+		this.alias = system.alias;
+		this.oauthConfig = system.oauthConfig;
+	}
+
+	@ApiProperty({
+		description: 'Flag to request only systems with oauth-config.',
+		required: false,
+		nullable: true,
+	})
+	type: string;
```

Comment:
I found a boolean param here https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/news/controller/dto/filter-news.params.ts#L30
maybe that helps you